specification/cognitiveservices/data-plane/TextTranslation/V3/TextTranslationV3.json

```diff
@@ -0,0 +1,930 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "Translator Text API",
```

Comment:
Title will be used for client class name
Please change to "Translator Text Client" to keep consistent with other Cognitive Services.